packages/i18n/README.md

```diff
@@ -129,41 +101,91 @@ Just import the `t` function and use it to translate the key.
 ```typescript
 import { t } from '@extension/i18n';
 
-console.log(t('helloWorld')); // Hello, World!
+console.log(t('loading')); // Loading...
 ```
 
 ```typescript jsx
 import { t } from '@extension/i18n';
 
 const Component = () => {
   return (
-    <div>
-      {t('helloWorld')} // Hello, World!
-    </div>
+    <button>
+      {t('toggleTheme')} // Toggle Theme
+    </button>
   );
 };
 ```
 
 ### Placeholders
 
-If you want to use placeholders, you can pass the second argument as a string or an array.
+If you want to use placeholders, you can use the following format.
+
+> For more information, see the [Message Placeholders](https://developer.chrome.com/docs/extensions/how-to/ui/localization-message-formats#placeholders) section.
+
+`locales/en/messages.json`
+
+
+```json
+{
+  "greeting": {
```

Comment:
IMO, it's more important to create example for this key, than for ```hello``` and how to use this 'placeholders'.

Maybe create 2 examples.

Reply:
Thx! I added example.

[feat: Add placeholder example](https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/613/commits/fa0a41b2f5fcd7497809745a91823ebec36b1317)